subsys/bluetooth/mesh/light_ctrl_internal.h

```diff
@@ -23,7 +23,7 @@ extern "C" {
 #endif
 
 static inline const struct bt_mesh_sensor_format *
-prop_format_get(enum bt_mesh_light_ctrl_prop id)
+prop_format_get(int id)
```

Comment:
Why not just extend the enum instead?

Reply:
bt_mesh_light_ctrl_prop and bt_mesh_light_ctrl_coeff are used in stack generally. Extending the bt_mesh_light_ctrl_prop seems likely to create a lot changes. And I don't see any issue that way the api is internal enough to be not confused.

Reply:
This API becomes public in #13899 . But I see that PR makes the same change. However, I don't see how adding values to an enum definition will cause any changes at all? As long as old values stay the same and no code uses the new definitions, nothing will "magically" change just because extra values are added.

Then again, we have `properties.h` which defines constants for all of these. I think I agree to change this to an `unit16_t`, and then in a new task just remove the `enum bt_mesh_light_ctrl_prop` entirely and use constants from `properties.h` instead in the Light Ctrl model implementations. It's weird that this is doubly defined.